pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -3938,7 +3938,11 @@ void FullscreenUI::DrawGraphicsSettingsPage(SettingsInterface* bsi, bool show_ad
 			s_tv_shaders, std::size(s_tv_shaders), true);
 	}
 
-	static constexpr const char* s_gsdump_compression[] = {FSUI_NSTR("Uncompressed"), FSUI_NSTR("LZMA (xz)"), FSUI_NSTR("Zstandard (zst)")};
+	static constexpr const char* s_gsdump_compression[] = {
+		FSUI_NSTR("Uncompressed"), 
```

Comment:
You have a space at the end of the line.

Reply:
Fixed.